btr-web/btr-common-components/utils/isoCountriesList.ts

```diff
@@ -2031,3 +2031,16 @@ export const iscCountriesListSortedByName = isoCountriesList.toSorted(
     return 0
   }
 )
+
+// three options that will be placed on the top of the citizenship dropdown menu
+const canadaUS: Array<BtrCountryI> = [
+  { name: 'Canada (Citizen)', alpha_2: 'CA' },
+  { name: 'Canada (Permanent Resident)', alpha_2: 'CA_PR' },
+  { name: 'United States', alpha_2: 'US' }
+]
+
+const otherCountries: Array<BtrCountryI> = iscCountriesListSortedByName.filter(
+  country => country.alpha_2 !== 'CA' && country.alpha_2 !== 'US'
+)
```

Comment:
It would be good to have the list sorted here, with order of display and all. Downside would be if we want to add any visual parts for the grouping (the thing to consider before deciding where to re-sort)